keyboards/primekb/prime_e/keymaps/madhatter/keymap.c

```diff
@@ -96,17 +85,13 @@ void matrix_scan_user(void) {
 
 //function for layer indicator LED
 layer_state_t layer_state_set_user(layer_state_t state) {
+    writePin(CAPS_LED_PIN, layer_state_cmp(state, 0));
     if (layer_state_cmp(state, 1)) {
```

Comment:
The same can be done here, and for layer 2, as well.

Reply:
done